src/app/screens/connectors/ConnectSpark/index.tsx

```diff
@@ -22,10 +23,16 @@ export default function ConnectSpark() {
   async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
     event.preventDefault();
     setLoading(true);
+
+    const mnemonic = await api.generateMnemonic();
+    const encryptedMnemonic = await api.encryptMnemonic(mnemonic);
```

Comment:
is this OK? I saw we have an existing `setMnemonic` function which will also derive the nostr key

Reply:
yess! at this stage there is no account stored in db. so we can't use that script

Reply:
But what about the nostr key? do we need to do anything there?